config/default.yml

```
@@ -1096,6 +1096,11 @@ Rails/UnusedIgnoredColumns:
   Include:
     - app/models/**/*.rb
 
+Rails/UnusedRenderContent:
+  Description: 'Do not specify body content for a response with a non-content status code.'
+  Enabled: pending
```

Can you add `Severity: warning` because this looks like a Lint-equivalent rule.

```suggestion
  Severity: warning
```